### DISCUSSION TITLE
[吐槽]喜欢我纯御驱动盘么

### DISCUSSION THREAD

**Discussion by LShanShi** [share121/inter-knot#3245] [其他]
两眼一黑,音像店的调律产出这么逆天??可以猜猜是什么驱动盘
![微信图片_20240820091113](https://github.com/user-attachments/assets/99dbe26a-15ea-4dd3-8744-c2805943e989)

**Comment by Jshuya24** [comment]
我的更逆天
![9D3AC69C089965B8013176B7581D9876](https://github.com/user-attachments/assets/c63141f0-e349-4c02-9714-39c26db21cd3)

**Reply by I-m-Miss-Lucy-s-dog** [reply]
不是 你這連主詞條都沒用處阿

**Reply by Jshuya24** [reply]
打出来直接是隔壁的融化伤害
